tests/test_modules/test_builtin/test_childpart.py

```diff
@@ -105,6 +105,7 @@ def test_init(self):
         assert info_out.name == "sourceportConnector"
         assert info_out.port == Port.INT32
         assert info_out.connected_value == "Connector1"
+        assert self.c.block_view().layout.value.visible == [True, True, True]
```

Comment:
This test fails on master and passes here, so this should be sufficient (with the initial_visibility=True removed above)